streams/tr.m3u

```diff
@@ -554,6 +556,8 @@ https://live.artidijitalmedya.com/artidijital_vuslattv/vuslattv/playlist.m3u8
 https://embedlp.becdn.net/womantv.m3u8
 #EXTINF:-1 tvg-id="WorldTurkTV.tr",World Turk TV (1080p)
 https://live.artidijitalmedya.com/artidijital_worldturk/worldturk/playlist.m3u8
+#EXTINF:-1 tvg-id="YabanTV.tr",Yaban TV (720p)
+https://yayin1.canlitv.fun/canlitv/yabantv.stream/playlist.m3u8?hash=f4ae19f6782b8755f8516677d81d5198
```

Comment:
geo-blocked

Reply:
It could be a geoblock. There is no problem with the hash part. The broadcast belongs to that hash. @Dum4G @BellezaEmporium 

![image](https://github.com/user-attachments/assets/abfe3797-8d76-40e5-9597-ded1addf8f64)

It gives a certificate error, the image appears only after I approve it. It gives a warning titled "Site is not secure".

![image](https://github.com/user-attachments/assets/adf72a48-71bf-4543-bddf-8fb13cf1c9c2)

![image](https://github.com/user-attachments/assets/029450dc-b0f5-406e-a54e-6ece952699dc)

Reply:
https://livepush.io/hlsplayer/index.html It works when I tested it here too.

![image](https://github.com/user-attachments/assets/b57e1bf6-14d5-403e-80ce-a1364151cba8)

Reply:
When I checked with Japan location VPN I got 403 error. Geo-blocked

![image](https://github.com/user-attachments/assets/a0610815-b4dc-4112-8ea1-ff2e0f2c1e8c)

Reply:
New PR opened. https://github.com/iptv-org/iptv/pull/17819 @Dum4G @BellezaEmporium